addons/account/models/account_move.py

```diff
@@ -2954,6 +2958,40 @@ def _compute_balance(self):
         for line in self:
             line.balance = line.debit - line.credit
 
+    @api.model
+    def search_read(self, domain=None, fields=None, offset=0, limit=None, order=None):
+        # Add the domain and order by in order to compute the cumulated balance in _compute_cumulated_balance
+        return super(AccountMoveLine, self.with_context(domain=domain or [], order_by=order)).search_read(domain, fields, offset, limit, order)
+
+    def _compute_cumulated_balance(self):
+        # invert the order by
+        order_by = [x.strip().split(' ') for x in (self.env.context['order_by'] or self._order).split(',')]
```

Comment:
Not the same thing. The value for that key can be Falsy. In that case, the ORM takes the default order of the model, and that's what I'm doing here too.
I used `get()` to avoid a `KeyError` but if it happens, the values computed will be wrong...